subprojects/core-api/src/main/java/org/gradle/process/JavaDebugOptions.java

```diff
@@ -31,6 +33,16 @@ public interface JavaDebugOptions {
      */
     @Input Property<Boolean> getEnabled();
 
+    /**
+     * The address of the debugger server.
```

Comment:
That sentence is weird.
What about:
> The host on which to listen for the debugger.

I would also add that `*` can be used to listen on all interfaces.

Reply:
I would actually copy the content you have in the documentation for the properties.

Reply:
@ljacomet Done, please take a look.

Reply:
Since this host name is used both in the server mode and in the client mode, it can mean either the host to listen on or the host to connect to, so mentioning just the host on which to listen doesn't feel enough. I followed your suggestion and copied the property docs here. Assuming that you approved that one, I'm merging this now.